docs/user-guide/gpudeduplication.rst

```diff
@@ -5,164 +5,367 @@
+  - Setting the ``false_positive_check`` flag to ``False`` is ideal for optimal performance.
+  - Clear the ``cache_dir`` between runs to avoid data from previous runs interfering with the current run's results.
+
+""""""""""""
+CLI Utility
+""""""""""""
+
+.. caution::
+  Fuzzy deduplication CLI scripts only work with the specific ID format generated by the :code:`add_id` script. If the
+  dataset does not contain ID's in this format it's recommended to create them with the :code:`add_id` script as follows:
+
+  .. code-block:: bash
+
+    add_id \
+      --id-field-name="my_id" \
+      --input-data-dir=<Path to directory containing jsonl files> \
+      --id-prefix="doc_prefix" \
+      --log-dir=./log/add_id
+
+  This will create a new field named :code:`my_id` within each json document which will have the form "doc_prefix-000001".
+  If the dataset already has a unique ID this step can be skipped.
+
+Once a unique ID has been added to each document, users can proceed with fuzzy deduplication which roughly require the following
+steps (all scripts are included in the `nemo_curator/scripts/fuzzy_deduplication <https://github.com/NVIDIA/NeMo-Curator/blob/main/nemo_curator/scripts/fuzzy_deduplication>`_ subdirectory):
+
+1. Compute Minhashes
+  - Input: Data Directories
+  - Output: ``minhashes.parquet`` for each data dir.
+  - Example call:
+
+       .. code-block:: bash
+
+               # same as `python compute_minhashes.py`
+               gpu_compute_minhashes \
+                 --input-data-dirs /path/to/jsonl/dir1 /path/to/jsonl/dir2 \
+                 --output-minhash-dir /path/to/output_minhashes \
+                 --input-json-text-field text_column_name \
+                 --input-json-id-field id_column_name \
+                 --minhash-length number_of_hashes \
+                 --char-ngram char_ngram_size \
+                 --hash-bytes 4(or 8 byte hashes) \
+                 --seed 42 \
+                 --log-dir ./
+                 # --scheduler-file /path/to/file.json
+
+.. _fuzzydup_lsh:
+
+2. Buckets (Minhash Buckets)
+  - Input: Minhash directories
+  - Output: ``_buckets.parquet``
+  - Example call:
+
+       .. code-block:: bash
+
+               # same as `python minhash_lsh.py`
+               minhash_buckets \
+                 --input-data-dirs /path/to/output_minhashes/dir1 /path/to/output_minhashes/dir2 \
+                 --output-bucket-dir /path/to/dedup_output \
+                 --input-minhash-field _minhash_signature \
+                 --input-json-id-field id_column_name \
+                 --minhash-length number_of_hashes \
+                 --num-bands num_bands \
+                 --buckets-per-shuffle 1 `#Value b/w [1-num_bands]. Higher is better but might lead to oom` \
+                 --log-dir ./
+                 # --scheduler-file /path/to/file.json
+
+3. False positive check (optional): If skipping this step, proceed to the :ref:`skip fp check section <fuzzydup_nofp>`.
```

Comment:
```suggestion
3. False Positive Check (optional): If skipping this step, proceed to the :ref:`skip fp check section <fuzzydup_nofp>`.
```
Also :ref:`skip fp check section <fuzzydup_nofp>` displays strangely?

Reply:
The final build skips below after `Jaccard Compute` to the `Buckets to Edgelist` section.